web-test-runner.config.mjs

```diff
@@ -60,7 +59,6 @@ export default {
   testsStartTimeout: 600000, // 10 minutes
   testsFinishTimeout: 600000, // 10 minutes
   plugins: [
-    esbuildPlugin({ ts: true }), // Workaround: prevent the issue "Could not import module ..." (randomly)
```

Comment:
I don't think this change is related to the PR.

Reply:
The `esbuildPlugin` should be removed because the root cause of the problem (in the comment above) is async/await incorrect